public/data/merouanezouaid.json

```diff
@@ -0,0 +1,22 @@
+{
+    "name": "Merouane Zouaid",
+    "bio": "Hello, I am Merouane Zouaid, a sophomore majoring in Computer Engineering at the Higher School of Technology of Agadir. Very passionate about Software Engineering, Web Development, Artificial Intelligence, and Obsessed with Open Source :) .",
+    "avatar": "https://github.com/merouanezouaid.png",
+    "links": [
+      {
```

Comment:
Maybe make a separate PR for adding your profile?

Reply:
Done!